docs/02-app/02-api-reference/05-next-config-js/redirects.mdx

```diff
@@ -38,8 +38,6 @@ module.exports = {
 
 Redirects are checked before the filesystem which includes pages and `/public` files.
 
-Redirects are not applied to client-side routing (`Link`, `router.push`), unless [Middleware](/docs/app/building-your-application/routing/middleware) is present and matches the path.
```

Comment:
Let's add a note to say this only applies to the Pages Router, not delete it. I don't make permission to suggest changes to your fork.

Reply:
I agree, removing wasn't the best idea especially when we can just show this sentence solely for Pages Router with `<PagesOnly />`.

I updated the text, hope this does it :)